examples/passport-login/src/authentication-strategies/facebook.ts

```diff
@@ -3,16 +3,15 @@
 // This file is licensed under the MIT License.
 // License text available at https://opensource.org/licenses/MIT
 
-import {asAuthStrategy, AuthenticationStrategy} from '@loopback/authentication';
-import {StrategyAdapter} from '@loopback/authentication-passport';
-import {Strategy} from 'passport-facebook';
-import {bind, inject} from '@loopback/context';
-import {extensionFor} from '@loopback/core';
-import {UserProfile} from '@loopback/security';
-import {User} from '../models';
-import {Request, RedirectRoute} from '@loopback/rest';
-import {PassportAuthenticationBindings} from './types';
-import {mapProfile} from './types';
+import { asAuthStrategy, AuthenticationStrategy } from '@loopback/authentication';
+import { StrategyAdapter } from '@loopback/authentication-passport';
+import { bind, inject } from '@loopback/context';
+import { extensionFor } from '@loopback/core';
+import { RedirectRoute, Request } from '@loopback/rest';
+import { UserProfile } from '@loopback/security';
+import { Strategy } from 'passport-facebook';
+import { User } from '../models';
+import { mapProfile, PassportAuthenticationBindings } from './types';
```

Comment:
This is not following our formatting style, did you forgot to run prettier after re-organizing the imports?

Reply:
Probably. Let me reformat.